app/services/live_config/rule_evaluator.py

```
@@ -0,0 +1,522 @@
+
+            if flag_data and isinstance(flag_data, dict):
+                targets = flag_data.get("targets", [])
+
+                for target in targets:
+                    audience = target.get("audience")
+                    if audience and user_matches_audience(
+                        audience, user_id, user_email, custom_data
+                    ):
+                        matched_targeting = True
+                        break
+    except:
```

Ruff (0.14.8)

Do not use bare `except`

(E722)